File: src/main/java/com/mallang/auth/infrastructure/oauth/github/dto/GithubTokenRequest.java
package com.mallang.auth.infrastructure.oauth.github.dto;

import com.fasterxml.jackson.databind.PropertyNamingStrategies.SnakeCaseStrategy;
import com.fasterxml.jackson.databind.annotation.JsonNaming;

@JsonNaming(SnakeCaseStrategy.class)
public record GithubTokenRequest(
        String clientId,
        String clientSecret,
        String code,
        String redirectUri
) {
}

File: src/main/kotlin/com/mallang/auth/infrastructure/oauth/github/dto/GithubTokenRequest.kt
package com.mallang.auth.infrastructure.oauth.github.dto

import com.fasterxml.jackson.databind.PropertyNamingStrategies
import com.fasterxml.jackson.databind.annotation.JsonNaming

@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy::class)
data class GithubTokenRequest(
        val clientId: String,
        val clientSecret: String,
        val code: String,
        val redirectUri: String
)
